### ISSUE TITLE
MetaDynamics ZeroDivisionError for bias that is exactly equal to 1

### ISSUE THREAD

**Issue by CoryKornowicz** [openmm/openmm#3633]
https://github.com/openmm/openmm/blob/dbb4827e6569c0c1b2eb42ad9bd40e17b400a40a/wrappers/python/openmm/app/metadynamics.py#L108

This line dictates the bias must be greater than or equal to one, but when this line is exectuted 

https://github.com/openmm/openmm/blob/dbb4827e6569c0c1b2eb42ad9bd40e17b400a40a/wrappers/python/openmm/app/metadynamics.py#L127

The self._deltaT is set to 0 and that causes 
```
height = self.height*np.exp(-energy/(unit.MOLAR_GAS_CONSTANT_R*self._deltaT))
```
to through a ZeroDivisionError. 

Thoughts on setting line 108 to 
```
 if biasFactor <= 1.0: 
```
to avoid the issue of 0 division?

**Comment by peastman** [comment]
Good catch!  You're exactly right.  A PR would be welcome!

**Comment by peastman** [comment]
Fixed by #3636.
